.github/workflows/lagom-template.yml

```diff
@@ -59,6 +60,13 @@ jobs:
           serenity_ccache_path: ${{ env.CCACHE_DIR }}
           download_cache_path: ${{ github.workspace }}/Build/caches
 
+      - name: Export VcPkg GitHub Actions cache environment variables
```

Comment:
not a big deal but I think the official casing is just "vcpkg", I don't see VcPkg used anywhere by microsoft

Reply:
actual review comment: Does it make sense for this step to go into caches.yml?